service/aesEncoder.go

```diff
@@ -0,0 +1,61 @@
+package service
+
+import (
+	"awesomeProject3/service/de_encoding"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/hex"
+	"io"
+)
+
+type aesEncoder struct {
+	key []byte
```

Comment:
Just a reminder that `key` should be part of `Encrypt/Decrypt` methods signature.